aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/CallbackContext.java

```diff
@@ -21,13 +22,18 @@ public class CallbackContext extends StdCallbackContext implements TaggingContex
     private boolean storageAllocated;
     private boolean allocatingStorage;
     private boolean readReplicaPromoted;
+    private boolean automaticBackupReplicationStopped;
+    private boolean automaticBackupReplicationStarted;
+    private String dbInstanceArn;
 
     private TaggingContext taggingContext;
-
+    private ProbingContext probingContext;
     private Map<String, Long> timestamps;
 
     public CallbackContext() {
         super();
+        this.probingContext = new ProbingContext();
+        this.probingContext.setProbingEnabled(true);
```

Comment:
AFAIU, Probing was used before for DBInstance modification as it had no instant impact on DBInstance status. Then it was removed after in stabilization rework #243 . Can we do smth similar here? To verify parameters are updated instead of probing?